src/activy/utils/stage/getStage.py

```
@@ -0,0 +1,80 @@
+    Extension of getStage that supports ignoring specific stages.
+    Returns the highest-scoring non-ignored stage, or None if no match is found.
+    """
+    # Convert ignored_stage to list if it's a single value
+    if ignored_stage is not None and not isinstance(ignored_stage, (list, tuple)):
+        ignored_stage = [str(ignored_stage)]
+    elif ignored_stage is not None:
+        ignored_stage = [str(item) for item in ignored_stage]
+    else:
+        ignored_stage = []
+
+    import cv2
```

Ruff (0.8.2)

Redefinition of unused `cv2` from line 1

Remove definition: `cv2`

(F811)